tltorch/factorized_layers/factorized_linear.py

```diff
@@ -2,6 +2,7 @@
 import numpy as np
 from torch import nn
 import torch
+import torch.utils.checkpoint as cp
```

Comment:
Let's be explicit for users, we can just use the full name:
```suggestion
from torch.utils import checkpoint
```

Reply:
Sure, checkpoint is better